### DISCUSSION TITLE
Vertical Sidebery Workflow + Bookmar Toolbar improvements

### DISCUSSION THREAD

**Discussion by KAGEYAM4** [Naezr/ShyFox#44] [Ideas]
Thank you for this great Firefox Theme. Removing Dependency of fx-autoconfig really made it more easier.

I use vertical panel and vertical pinned tab for sidebery. Also i have created a FeatureRequest on Sidebery if it get implemented, we will have global Pinned tabs and local panel pinned tabs ( maybe worth considering if it gets implemented in future so shyfox dosent need to do rework for compatibility ).

Screenshot->

My current firefox using [firefox-ui-fix ](https://github.com/black7375/Firefox-UI-Fix) ->
![image](https://github.com/Naezr/ShyFox/assets/75798544/1d4758c2-efee-4038-b709-247039c22e07)

Using this Sidebery setup in ShyFox, it looks like this ->
![image](https://github.com/Naezr/ShyFox/assets/75798544/1ad7a6b0-0121-47b5-a97f-7ecac50d129f)

Also if bookmark Toolbar can be improved, don't know exactly how it should look or work but just wanted to ask if you have some implemention in mind. I use bookmark toolbar for bookmarks that i access frequently or that has higher priority, rest of all other bookmar sits in bookmark menu. Screenshot ->
![image](https://github.com/Naezr/ShyFox/assets/75798544/4ac83b91-0f6d-46ea-b533-05c7120027b5)

For ShyFox, you can see how it looks in previous screenshot.

**Comment by Naezr** [comment]
About the sidebar. When importing styles, Sidebery does not delete the old style, but inserts the new one after the old one. To make the ShyFox style for Sidebery work, go to the Style editor page in Sidebery settings and delete everything from the text box on the right. Then re-import the file from ShyFox. 

About the bookmarks panel. Actually, after switching to vertical tabs, I started to use it very rarely. I put it there because I only use it to occasionally open rarely used sites and save them there. I organized them into four folders and they fit perfectly in the width of the sidebar. In your case, you could group all your folders into other folders, although it might be inconvenient. Sure, I could put the bookmarks bar under the navigation bar, but right now I'm running out of energy. I want to take a break from customizing Firefox for a while, because I've already created the perfect theme for myself.

**Comment by KAGEYAM4** [comment]
> About the sidebar. When importing styles, Sidebery does not delete the old style, but inserts the new one after the old one. To make the ShyFox style for Sidebery work, go to the Style editor page in Sidebery settings and delete everything from the text box on the right. Then re-import the file from ShyFox.

Got it , ` ColorScheme in Apperance needs to be set to firefox`. My settings were overwriting it to `dark`.

> right now I'm running out of energy. I want to take a break from customizing Firefox for a while, because I've already created the perfect theme for myself.

Totally, i just wanted to put it here, so sometime in future this can be implemented if you liked the idea.

**Comment by Naezr** [comment]
Okay, I get the idea, not everyone is comfortable having such a small bookmark bar. In the last version it was not supported at all, so now we have something. I'm aiming to add customization at the level of choosing the location of the navigation bar (top or bottom) or as you have now suggested the location of the bookmarks bar. But so far I haven't reached the level of modularity that you can just replace the conditional `shy-navbar.css` with the alternative `shy-navbar-bottom.css` without affecting the rest of the files. But I have a lot of room to grow, so it's all ahead of me.

**Comment by Naezr** [comment]
About Sidebery. You can use any style for Sidebery, not necessarily packed with the theme one. But I'm not responsible for how it will look, that's your problem
